packages/fxa-auth-server/lib/senders/emails/templates/newDeviceLogin/en.ftl

```diff
@@ -4,6 +4,8 @@ newDeviceLogin-subject = New sign-in to { $clientName }
 # Variables:
 # $clientName (String) - A client the user hasn't signed into before (e.g. Firefox, Sync)
 newDeviceLogin-title-2 = Your { -product-firefox-account } was used to sign in
+# $clientName (String) - A client the user hasn't signed into before (e.g. Firefox, Sync)
```

Comment:
I think comment is just extra copy/paste. There's no $clientName variable in the the `newDeviceLogin-title-3` string.

Reply:
Good catch! I'll remove from the original string too.